DatadogSessionReplay/SRSnapshotTests/SRFixtures/Sources/SRFixtures/Fixtures.swift

```diff
@@ -7,7 +7,11 @@
 import UIKit
 import SwiftUI
 
-public enum Fixture: CaseIterable {
+public protocol FixtureProtocol {
+    func instantiateViewController() -> UIViewController
+}
```

Comment:
We need to be able to inject some fixtures from `SRSnapshotTests`, to avoid adding dependencies to `SRFixtures`.